### PR TITLE
[4.0] helpsite removal

### PR DESCRIPTION
**Pull request by brianteeman**
Removes the remaining parts of the helpsite code that was removed in 3.9. due to file renaming these were missed.

code review

discovered by finally having the lint running

### CODE REVIEW

**Comment by Quy** [comment]
Delete `\administrator\help\helpsites.xml`?

**Comment by brianteeman** [comment]
No that is used by the help button

**Comment by Quy** [comment]
Deleted the file without affecting the Help button. The value is obtained from the configuration file.

`public $helpurl = 'https://help.joomla.org/proxy?keyref=Help{major}{minor}:{keyref}&lang={langcode}';`

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on a0f64ec614a0defde1c0d9607d22bd54d3c146ca<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28088">issues.joomla.org/tracker/joomla-cms/28088</a>.</sub>

**Comment by astridx** [comment]
If I am right, first PR was this: https://github.com/joomla/joomla-cms/pull/24927

I applied this patch and run `composer i` and `npm ci`.

I opened user manager and clicked Help (Aide). Help Site `https://help.joomla.org/proxy?keyref=Help40:Users_User_Manager&lang=fr` was opened.

I opened some more Help Sites and had no problems.

**Comment by astridx** [comment]
I have tested this item :white_check_mark: successfully on a0f64ec614a0defde1c0d9607d22bd54d3c146ca<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28088">issues.joomla.org/tracker/joomla-cms/28088</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28088">issues.joomla.org/tracker/joomla-cms/28088</a>.</sub>

File: libraries/classmap.php [hunk 1]
```diff
@@ -314,7 +314,6 @@
 JLoader::registerAlias('JFormFieldFrontend_Language',       '\\Joomla\\CMS\\Form\\Field\\FrontendlanguageField', '5.0');
 JLoader::registerAlias('JFormFieldGroupedList',             '\\Joomla\\CMS\\Form\\Field\\GroupedlistField', '5.0');
 JLoader::registerAlias('JFormFieldHeadertag',               '\\Joomla\\CMS\\Form\\Field\\HeadertagField', '5.0');
-JLoader::registerAlias('JFormFieldHelpsite',                '\\Joomla\\CMS\\Form\\Field\\HelpsiteField', '5.0');
```
Comment (wilsonge) [inline libraries/classmap.php:317]: Can we just ensure the deprecation property is updated in the 3.x branch for this file to be 4.0 please

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**